debian/st2chatops.service

```diff
@@ -14,6 +14,8 @@ TimeoutSec=60
 PrivateTmp=true
 Restart=on-failure
 RestartSec=5
+StandardOutput=syslog+console
+StandardError=syslog+console
```

Comment:
It does not. This directive instructs journalctl to write to console which should get logged to /var/log according to docs but it doesn't. So sad.